src/helix-ui/styles/components/lists.less

```diff
@@ -1,24 +1,61 @@
 @import (reference) 'components/grid';
 
-hx-def + hx-def {
+dl {
+  display: block;
+  margin: 0;
+}
+
+dl > div {
+  display: flex;
+}
+
+div + div {
```

Comment:
should be `dl > div + div` so that it doesn't match divs outside of a list  (see comment above)